### DISCUSSION TITLE
Request for Blender Shape key Customization

### DISCUSSION THREAD

**Discussion by deanvilgel** [cables-gl/cables_docs#133] [Feature Requests] [answered]
Hi! I was trying to add face varietion for my Model, which was done by shape key in blender. This Shape key feature are supported in blender or unity, and GLB file also seems to remember animation made with Shape key. 

When I brought my Model and Animation to Cables, This Shape key animation Seems to be broken. 

I know Facial Expression Could be done by bone animation, But Shape key can do this more easier than that.

Any Support For this would be really astonishing!

This is test animation for the Shape key. Bone animation seems to be working, but Shape key animation seems to be broken and only show themselves around 6 second Shortly.
[](https://cables.gl/p/qT5pKU)

I will Contain The original blender File  too,
[marshmellow.zip](https://github.com/cables-gl/cables_docs/files/7820327/marshmellow.zip)

**Comment by steam0r** [comment]
hey, i am adding this here for reference as i think it's your example: https://cables.gl/p/61d6a0dc73b7e81b0de63812

**Comment by deanvilgel** [comment, marked as answer]
> 

Thanks! I thought I uploaded link, but should have done sth wrong /:

**Reply by akerb00m** [reply]
Hi there, Very interesting, i have been trying to get acces to blender / Glb shapekeys in cables but i can't seem to get it to work... can i ask how you exported the shapekey from blender, baked to keyframes or as an NLA track maybe ?
cheers !

**Reply by steam0r** [reply]
this is handled/solved here, right? https://github.com/cables-gl/cables_docs/discussions/260

**Reply by akerb00m** [reply]
@steam0r 
i did notice that this way of getting a vertex animation from blender to cables does not seem to work when .GLB is compressed.
made a little test here https://cables.gl/edit/tq52vi
Cheers !
